src/hitable/Sky.cpp

```diff
@@ -10,30 +10,29 @@ using namespace CGL;
 using namespace std;
 using namespace cv;
 
-Sky::Sky(Mat texture, double radius) {
-    this->radius = radius;
+Sky::Sky(Mat texture, double radius) : textureMap(texture.cols, texture.rows) {
+    radius = radius;
```

Comment:
I accidentally overloaded the variable name `radius` here, are you sure this reassigns the struct attribute `radius` to the value of the parameter `radius`?

Reply:
I'm not sure how this works so i'll change it just in case.

Reply:
Actually you're right, it should be like `this->radius`